lnrpc/rpc.proto

```diff
@@ -290,72 +559,136 @@ message HTLC {
 }
 
 message ActiveChannel {
+    /// Whether this channel is active or not
     bool active = 1 [json_name = "active"];
+    /// The identity pubkey of the remote node
     string remote_pubkey = 2 [json_name = "remote_pubkey"];
+
+    /**
+    ChannelPoint is the outpoint (txid:index) of the funding transaction. With this value, Bob will be able to generate a signature for Alice's version of the commitment transaction.
```

Comment:
See comment below about including the name of the field. Also, why is this camel case rather than snake case when it seems that the field names are snake case?